HW3_Angela Chun.Rmd

```diff
@@ -0,0 +1,86 @@
+---
+title: "HW3"
+author: "Angela Chun"
+date: "7/19/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load libraries, include=FALSE}
+library("ggplot2")
+library("tidyverse")
```

Comment:
I think everyone is still getting the hang of this, myself included, but I think ggplot2, dplyr and tidyr are included in tidyverse.